apps/torch/app.js

```diff
@@ -11,14 +11,16 @@ Bangle.setLCDBrightness(1);
 Bangle.setLCDPower(1);
 Bangle.setLCDTimeout(0);
 g.reset();
+bgBackup = g.theme.bg;
 g.setTheme({bg:settings.bg,fg:"#000"});
 g.setColor(settings.bg);
 g.fillRect(0,0,g.getWidth(),g.getHeight());
 Bangle.setUI({
   mode : 'custom',
-  back : load, // B2: Clicking the hardware button or pressing upper left corner turns off (where red back button would be)
-  btn : (n)=>{ // B1: Any button turns off
-    if (process.env.HWVERSION==1 && (n==1 || n==2 || n==3)) {
+  back : ()=>{g.setTheme({bg:bgBackup});load();}, // B2: SW back button
```

Comment:
Is this needed? The theme should be reset every time a watch app loads?

I think this is a case of you using a non-standard firmware again. I think really rather than bodging all the apps, we should just add a fix to set the default theme in @halemmerich's loader?

Reply:
Yeah sorry, I tested it on both my usual setup and a factory reset and saw a little better experience on mine and no change in experience for standard, so I rationalized it that way. But it's gone now, I'll refrain from using halemmerich's stuff for a while as to not have this happen in PRs any more...